icu4c/source/i18n/plurrule.cpp

```diff
@@ -1618,7 +1618,7 @@ FixedDecimal::FixedDecimal(double n, int32_t v, int64_t f, int32_t e) {
     init(n, v, f, e);
     // check values. TODO make into unit test.
     //            
-    //            long visiblePower = (int) Math.pow(10, v);
+    //            long visiblePower = (int) Math.pow(10.0, v);
```

Comment:
this is a comment

Reply:
I know that it doesn't make any difference, but it makes sense to keep the code consistent even in the comments in case one day someone decides to uncomment those lines again. That said, if you prefer this not to be patched, I can remove it from the PR.